mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -52,12 +52,22 @@ class KMeans private (
    */
   def this() = this(2, 20, 1, KMeans.K_MEANS_PARALLEL, 5, 1e-4, Utils.random.nextLong())
 
+  /**
+   * Number of clusters to create (k).
```

Comment:
Nit: You could prefix this with `@return`
Interesting question: naming getters `getFoo` is of course standard in Java, and seems just fine too, although in Scala we'd be more used to accessing `foo` only. It's possible to declare `private var _foo` and then declare the public getter to have name `foo` here. Does anyone prefer that?